assets/thanos-querier/deployment.yaml

```diff
@@ -2,32 +2,54 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   labels:
-    app.kubernetes.io/name: thanos-querier
+    app.kubernetes.io/component: query-layer
+    app.kubernetes.io/instance: thanos-querier
+    app.kubernetes.io/name: thanos-query
```

Comment:
deployment name is still thanos-querier, I expect the name to reflect that, wdyt?

Reply:
name is the kubernetes app name, as kube-thanos is the defining entity here, I don't think we should change this, we should however eventually rename thanos-querier to thanos-query

Reply:
> we should however eventually rename thanos-querier to thanos-query

Agreed, can you open jira for this? Thanks! :)